pandas/_libs/tslibs/offsets.pyx

```diff
@@ -1586,8 +1586,8 @@ class DateOffset(RelativeDeltaOffset, metaclass=OffsetMeta):
     Besides, adding a DateOffsets specified by the singular form of the date
     component can be used to replace certain component of the timestamp.
 
-    Parameters
-    ----------
+    Parameters ''' numpydoc ignore=PR02 '''
+    ---------- ''' numpydoc ignore=PR02 '''
```

Comment:
I think I'd prefer to have the ignore in the code_checks rather than the docstring, but no strong opinion.

Reply:
yeah I'm in agreement, these changes turned out to fail some of the CI checks, and I've been unable to get the inline comments to pass all the checks. Until we can come up with a solution to resolve the PR02s I am in agreement we should leave all the "ignore" related code in code_checks.sh